tools/tpm2_quote.c

```diff
@@ -406,13 +490,13 @@ int execute_tool (int argc, char *argv[], char *envp[], common_opts_t *opts,
              }
              is_auth_session = true;
              break;
-       case ':':
+        case ':':
```

Comment:
unrelated change, separate patch for this and line 499.

Reply:
I can do this as part of a general rebase once the rest is conforming.